modules/rustbitcoin/module.cpp

```diff
@@ -17,16 +17,18 @@ namespace bitcoinfuzz
             return result;
         }
 
-        std::optional<std::vector<bool>> Rustbitcoin::deserialize_block(std::span<const uint8_t> buffer) const
+        std::optional<std::string> Rustbitcoin::deserialize_block(std::span<const uint8_t> buffer) const
         {
             auto pointer{rust_bitcoin_des_block(buffer.data(), buffer.size())};
             std::string result(pointer);
             free_c_string(pointer);
-            if (result == "unsupported segwit version") {
+            if (result == "unsupported segwit version" || result == "false" || result.empty()) {
                 return std::nullopt;
             }
```

Comment:
```suggestion
            if (result == "unsupported segwit version") {
                return std::nullopt;
            }
```

This is the only case that we will skip